docs/root/operations/cli.rst

```diff
@@ -43,11 +43,17 @@ following are the command line options that Envoy supports.
   *(optional)* One of the operating modes for Envoy:
 
   * ``serve``: *(default)* Validate the JSON configuration and then serve traffic normally.
-
+  * ``init_only``: Validate and initialize everything, then exit without serving traffic.
   * ``validate``: Validate the JSON configuration and then exit, printing either an "OK" message (in
     which case the exit code is 0) or any errors generated by the configuration file (exit code 1).
-    No network traffic is generated, and the hot restart process is not performed, so no other Envoy
+    Validation succeeds if the configuration and all required files are loaded successfully. No
+    network traffic is generated, and the hot restart process is not performed, so no other Envoy
     process on the machine will be disturbed.
+  * ``load``: Load only the provided JSON configuration file and then exit, printing either an "OK"
+    message (in which case the exit code is 0) or any errors generated by the configuration file
+    (exit code 1). Validation succeeds if the bootstrap JSON file is loaded successfully, even if
+    other resources are not available. No network traffic is generated, and the hot restart process
```

Comment:
Could you explicitly mention that missing filesystem paths are ignored during "load"?